instruments/toptica/topmode.py

```diff
@@ -310,7 +342,15 @@ def reference(self, param):
         :return: Response to the reference request
         :rtype: `str`
         """
-        return self.query("(param-ref '{})".format(param))
+        # the toptica topmode termination character is \r\n,
+        # but unfortunately serial communicator only allows for single
+        # character terminators.
```

Comment:
Note that in the develop branch there is support for multi-character terminators.